package.json

```diff
@@ -48,7 +48,7 @@
     "@babel/runtime": "^7.12.5"
   },
   "devDependencies": {
-    "@testing-library/dom": "^10.0.0",
+    "@testing-library/dom": "https://pkg.csb.dev/testing-library/dom-testing-library/commit/8abda5aa/@testing-library/dom",
```

Comment:
Needs https://github.com/testing-library/dom-testing-library/pull/1369 before we can merge this